### DISCUSSION TITLE
Can I translate FXGL 11 into Simple-Chinese?

### DISCUSSION THREAD

**Discussion by osiIiso** [AlmasB/FXGL#1211] [Q&A]
Can I translate this article into Chinese?
link:
https://github.com/AlmasB/FXGL/wiki/FXGL-11

I am a java developer and I am currently learning the FXGL11 framework, and, I would like to translate it into Chinese
If you agree to my request, I would like to put my translated article in your github description so that more Chinese java game developers can learn it.

**Comment by AlmasB** [comment]
Yes, sure, translations are welcome.

When you say "your github description", I'm not sure where you'd like to upload it. There's a dedicated "translations" page, so I was wondering if we could add it there? https://github.com/AlmasB/FXGL/wiki/Translations

**Reply by osiIiso** [reply]
Sorry I didn't make myself clear earlier, what I meant was that I wanted to translate your FXGL-11 instructions and example article into Chinese and provide it to you, the article is this
https://github.com/AlmasB/FXGL/wiki/FXGL-11
The following picture is what I am doing
![image](https://user-images.githubusercontent.com/53078833/202053565-b072abb0-5d23-4ee6-a589-cf421340a73c.png)

**Reply by AlmasB** [reply]
Yep that's fine, thank you, let me know if anything is needed from my side.

**Reply by osiIiso** [reply]
> Yep that's fine, thank you, let me know if anything is needed from my side.
hey,I found that the mario sample code has been deleted, can you provide me with the code ?
![image](https://user-images.githubusercontent.com/53078833/203487314-aafe52b6-785e-40b9-b07c-99f06de63bed.png)

https://github.com/AlmasB/FXGLGames/blob/master/Mario/src/main/java/com/almasb/fxglgames/mario/components/PlayerComponent.java
